openvdb/python/CMakeLists.txt

```diff
@@ -49,8 +52,14 @@ set(OPENVDB_PYTHON_INCLUDES)
 # the minimum python version is MINIMUM_PYTHON_VERSION - however this cannot
 # be provided to find_package(Python) with differing major versions. e.g.
 # calls to find_package(Python 2.7) fails if python3 is found on the system.
-function(OPENVDB_CHECK_PYTHON_VERSION _PY_VERSION _PY_PATH)
-  if(_PY_VERSION VERSION_LESS MINIMUM_PYTHON_VERSION)
```

Comment:
Made these arguments optional to handled the case where `_PY_VERSION` isn't defined due to find failure